helix-term/src/ui/picker.rs

```diff
@@ -330,13 +338,15 @@ pub struct Picker<T: Item> {
     show_preview: bool,
 
     callback_fn: Box<dyn Fn(&mut Context, &T, Action)>,
+    key_event_callback_fn: KeyEventCallback<T>,
```

Comment:
maybe make it optional? like `Option<KeyEventCallback<T>>`?

Reply:
Moved things around, the key_event_callback is not longer part of the constructor, instead there is a separate function to set the key event callback function.